__tests__/data/salesforce_router_metadata_input.json

```
@@ -0,0 +1,98 @@
+{
+  "batchedRequest": {
+    "version": "1",
+    "type": "REST",
+    "method": "POST",
+    "endpoint": "https://ap15.salesforce.com/services/data/v50.0/sobjects/Lead",
+    "headers": {
+      "Content-Type": "application/json",
+      "Authorization": "Bearer 00D2v000002lXbX!ARcA12345.MbUdtmlREscrN_nmytokenAzNkhT1XZIF0yPqyCQSnezWO3osqwerty12O7q41E9.LvedWY"
+    },
+    "params": {},
+    "body": {
+      "JSON": {
+        "Email": "peter.gibbons@initech.com",
+        "Phone": "570-690-4150",
+        "Rating": "Hot",
+        "Title": "VP of Derp",
+        "FirstName": "Peter",
+        "LastName": "Gibbons",
+        "PostalCode": "94115",
+        "City": "east greenwich",
+        "Country": "USA",
+        "State": "California",
+        "Street": "19123 forest lane",
+        "Company": "Initech"
+      },
+      "XML": {},
+      "JSON_ARRAY": {},
+      "FORM": {}
+    },
+    "files": {}
+  },
+  "metadata": [
+    {
+      "jobId": 1
+    },
+    {
+      "jobId": 1
```

Can you update these jobIds appropriately ?
```suggestion
{
      "jobId": 1
    },
    {
      "jobId": 2
      },
      {
      "jobId": 3
      }
```

---

done